doc/release/RELEASE-NOTES.md

```
@@ -22,7 +22,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe code/API changes here.*
 
 ### Architecture/Environment changes
-* *Describe architecture/environment changes here.*
+* [OSDEV-2083](https://opensupplyhub.atlassian.net/browse/OSDEV-2083) - The following updates were made:
+    * Updated Python version from `3.7` to `3.8` to support higher OpenSearch client version with necessary features.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)